fluent-package/msi/install-test.ps1

```diff
@@ -4,14 +4,30 @@ $msi = ((Get-Item "C:\\fluentd\\fluent-package\\msi\\repositories\\fluent-packag
 Write-Host "Installing ${msi} ..."
 
 Start-Process msiexec -ArgumentList "/i", $msi, "/quiet" -Wait -NoNewWindow
-Start-Service fluentdwinsvc
+$initialStatus = (Get-Service fluentdwinsvc).Status
+if ($initialStatus -ne "Stopped") {
+    Write-Host "The initial status must be 'Stopped', but it was '${initialStatus}'."
+    [Environment]::Exit(1)
+}
 
 $ENV:PATH="C:\\opt\\fluent\\bin;" + $ENV:PATH
 $ENV:PATH="C:\\opt\\fluent;" + $ENV:PATH
 
 td-agent --version
 
-Start-Sleep 3
+Write-Host "Measuring times to start the service..."
+$timeSpans = 0..2 | % {
+    Measure-Command { Start-Service fluentdwinsvc }
+    Start-Sleep 15 | Out-Null
+    Stop-Service fluentdwinsvc | Out-Null
+    Start-Sleep 15 | Out-Null
```

Comment:
I'm not sure `|Out-Null` is required.
I thought no Out-Null affects the result of $timeSpans, but it seems not.


```
Start-Sleep 15
Stop-Service fluentdwinsvc
Start-Sleep 15
```

Reply:
Oh! Really!
I want this for-each to collect only the results of `Measure-Command`, so I force other commands not to output any results.
However, if it does not affects the results, it may be unnecessary.
I will check it later.

Thanks!

Reply:
These commands do not output to the standard output, so we don't need `Out-Null` here!
I have removed them.
Thanks!